urbansearch/server/main.py

```diff
@@ -23,6 +24,12 @@ def __init__(self, run=True):
         self.app = Flask(__name__)
         self.register_blueprints()
 
+        @self.app.after_request
+        def apply_caching(response):
+            response.headers["Access-Control-Allow-Origin"] = "*"
+            response.headers["Access-Control-Allow-Headers"] = "Content-Type"
+            return response
+
         if run:
```

Comment:
Maybe add an if __name__ = "__main__" to be able to create and start server stand alone? Might be useful, but not absolutely necessary